spec/formatting.md

```diff
@@ -957,4 +957,20 @@ The _Default Bidi Strategy_ is defined as follows:
          prefix `fmt` with U+2068 FIRST STRONG ISOLATE
          and postfix it with U+2069 POP DIRECTIONAL ISOLATE.
 
+The auxiliary function `DIR`, which maps an _expression_ to one of
+« `'LTR'`, `'RTL'`, `'unknown'` », is expected to be defined using the
+_resolved value_ of the _expression_. An implementation can use
```

Comment:
Please use [semantic line breaks](https://sembr.org/), they make reviewing and editing markdown much easier.

Reply:
Tried to refine the placement of the line breaks; if there's anything specific, let me know.